packages/hash/frontend/tsconfig.json

```diff
@@ -1,12 +1,12 @@
 {
   "extends": "../../../tsconfig.base.json",
   "compilerOptions": {
+    "incremental": true,
     "isolatedModules": true,
     "jsx": "preserve",
     "lib": ["dom", "dom.iterable", "esnext"],
-    "target": "es5",
```

Comment:
Discussing this on Slack, because I'm not sure why this is here.

Reply:
Internal [slack thread](https://hashintel.slack.com/archives/C022217GAHF/p1645186873448139) for reference.

Reply:
Reverted in https://github.com/hashintel/hash/pull/320/commits/796c295d06c9917ce0135d1269c11c9bff0cd906

Reply:
Thanks @kachkaev